pkg/loadbalancer/loadbalancer.go

```diff
@@ -475,17 +475,21 @@ func (a *L3n4Addr) DeepCopy() *L3n4Addr {
 	}
 }
 
-// Hash calculates L3n4Addr's internal SHA256Sum.
+// Hash calculates a unique string of the L3n4Addr e.g for use as a key in maps
```

Comment:
At that point would it make sense to rename the function from `Hash` to something like `Key`?

Reply:
For ease of potential back-porting, I'll leave this as is for now.